admin-dev/themes/new-theme/js/pages/product/edit/product-partial-updater.ts

```diff
@@ -108,7 +108,11 @@ export default class ProductPartialUpdater {
     this.initialData = this.getFormDataAsObject();
     this.$productForm.submit(() => this.updatePartialForm());
     // 'dp.change' event allows tracking datepicker input changes
-    this.$productForm.on('keyup change dp.change', ':input', () => this.updateFooterButtonStates());
+    this.$productForm.on('keyup change dp.change',
+      // listen for all inputs except combination filters
+      `:input[name!="${ProductMap.combinations.list.attributeFilterInputName}"]`,
```

Comment:
Hmm, this is weird though 🤔 Does it also mean the input from combination list trigger the product footer update?